### PR TITLE
GitHub Action workflow to detect non accessible program urls

### PR DESCRIPTION
**Pull request by xm1k3**

### CODE REVIEW

File: .github/workflows/domains-validation.yml [hunk 1]
```diff
@@ -1,52 +1,85 @@
 name: Domains validation
-
-on:
+'on':
```
Comment (Mzack9999) [inline .github/workflows/domains-validation.yml:2]: Shouldn't this work with just `on`?

File: .github/workflows/domains-validation.yml [hunk 2]
```diff
@@ -1,52 +1,85 @@
...
   pull_request:
     branches:
       - main
-  workflow_dispatch:
-
+  workflow_dispatch: null
```
Comment (Mzack9999) [inline .github/workflows/domains-validation.yml:6]: I think also here leaving it empty should do the trick

File: .github/workflows/domains-validation.yml [hunk 3]
```diff
@@ -1,52 +1,85 @@
...
+      - name: Installing Validate domains binary
+        run: >
+          go install
+          github.com/projectdiscovery/public-bugbounty-programs/cmd/httpx-tester
+        shell: bash
+      - name: Check domains validation
+        run: |
+          httpx-tester -file urls.txt
+        shell: bash
+      - name: null
```
Comment (Mzack9999) [inline .github/workflows/domains-validation.yml:80]: Let's use more meaningful action names (eg. `checking URL availability`)

File: .github/workflows/httpx-check-cron.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      -- name: Installing Validate domains binary
+        run: >
+          go install
+          github.com/projectdiscovery/public-bugbounty-programs/cmd/httpx-tester
+        shell: bash
+      - name: Check domains validation
+        run: |
+          httpx-tester -file urls.txt
+        shell: bash
+      - name: null
```
Comment (Mzack9999) [inline .github/workflows/httpx-check-cron.yaml:34]: same as above

**Review by Mzack9999 — CHANGES_REQUESTED** [review]
suggesting minor changes related to naming

**Review by Mzack9999 — APPROVED** [review]

**Review by ehsandeep — APPROVED** [review]

**Pull request merged**